### ISSUE TITLE
Support for more languages

### ISSUE THREAD

**Issue by barbolo** [GoogleCloudPlatform/typescript-recaptcha-password-check-helpers#4]
Hi, I was trying to write a Ruby version of this code, however there's a WASM binary that is used in the final encryption step.

Can you share more information about the encryption function? Will you only share parts of it through closed source execution binaries/libraries?

**Comment by maxloeffler** [comment]
Hey @barbolo, have you figured out a decent way to reverse the WASM file? I'm facing the same problems at the moment.

**Comment by barbolo** [comment]
I gave up using it. :/

**Comment by maxloeffler** [comment]
@barbolo, yeah me neither, so I started analyzing their [Java implementation](https://github.com/GoogleCloudPlatform/java-recaptcha-password-check-helpers) of this library. Instead of cryptic pre-compiled wasm files, it uses standard bouncy castle for crypto which makes it way easier to debug. With this I was able to recreate the functionality one-to-one in python, even though it was quite a journey. My project is not ready to be published, but hit me up if you want to talk ^^.

**Comment by barbolo** [comment]
@maxloeffler thanks for sharing your progress. Unfortunately, reverse engineering might not be an option for me due to uncertainty about license restrictions.
